src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -71,14 +71,6 @@ class TableView extends React.Component<TableViewProps> {
       if (prevField.width !== nextField.width) {
         changed.push('width');
       }
-
-      trackAnalyticsEvent({
-        eventKey: 'discover_v2.update_column',
-        eventName: 'Discoverv2: A column was updated',
-        updated_at_index: columnIndex,
-        changed,
```

Comment:
Can we remove the logic to compute `changed ` too?

Reply:
Good catch. https://github.com/getsentry/sentry/pull/18240/commits/b255ae7d1cce2a5a438bdf7ee9941cb607bf8bfb